lesson13/index.html

```diff
@@ -1,16 +1,24 @@
 <!DOCTYPE html> 
 <html lang="ja"> 
-<head> 
+  <head> 
     <meta charset="UTF-8"> 
     <meta http-equiv="X-UA-Compatible" content="IE=edge"> 
     <meta name="viewport" content="width=device-width, initial-scale=1.0"> 
     <title>js-lesson</title> 
     <script src = "main.js" defer></script> 
     <link rel="stylesheet" href="css/style.css">
-</head> 
-<body> 
-    <button id="js-button">取得</button>
-    <ul id="js-ul"> 
-    </ul> 
-</body> 
+  </head> 
+  <body> 
+      <ul id="js-ul"></ul> 
+      <div id="js-modal" class="modal frame-out">
+        <p>ボタンを押してください</p>
+        <div class="button-container">
+          <button id="js-fetch-button" >取　得</button> 

```

Comment:
Q.
typoでしょうか？
デザイン的に、余白あけたいからスペース入れたのであれば、CSSで調整すべきだと感じました。
```suggestion
          <button id="js-fetch-button" >取得</button> 

```